lib/security/aes.js

```diff
@@ -0,0 +1,42 @@
+var CryptoJS = require("crypto-js");
+var q = require('q');
+
+/**
+* Encrypt an Object using AES with a specified secret key.
+* @param data - the data to encrypt.
+* @param key - the secret key to encrypt with.
+*/
+function encrypt(data, key) {
+  var deferred = q.defer();
```

Comment:
Just a heads up. Typically _q.defer()_ is viewed as an anti-pattern in the Promise world. Better to do:

```
return new Promise((resolve, reject) => {
  if (data === null || key === null) {
    reject(new Error('Data or Secret key is Null'))
  } else {
    resolve(CryptoJS.AES.encrypt(JSON.stringify(data), key))
  }
});
```